docs/cloud/webhooks.md

```diff
@@ -0,0 +1,287 @@
+Cloud](/cloud/connecting#manually-configure-prefect-api-settings).
+
+### Via the Prefect CLI
+
+_More information about using the Prefect CLI to manage webhooks coming soon_
+
+## Webhook endpoints
+
+The webhook endpoints have randomly generated opaque URLs that do not divulge any
+information about your Prefect Cloud workspace.  They are rooted at
+`https://api.prefect.cloud/hooks/`, (for example,
+`https://api.prefect.cloud/hooks/AERylZ_uewzpDx-8fcweHQ`).  Prefect Cloud assigns this
+URL when you create a webhook, and it may not be set via the API.  If you lose
+confidence that your webhook URL is confidential, you may rotate it at any time without
+losing the associated configuration.
+
+All webhooks may accept requests via the most common HTTP methods:
+
+* `GET`, `HEAD`, `DELETE` may be used for webhooks that define a static event template,
+  or a template that does not depend on the _body_ of the HTTP request.  The headers of
+  the request will be available for templates.
+* `POST`, `PUT`, `PATCH` may be used when the webhook request will include a body.  See
+  [How HTTP request components are handled](#how-http-request-components-are-handled)
+  for more details on how the body is parsed.
+
+Prefect Cloud webhooks are deliberately quiet to the outside world, and will only return
+a `204 No Content` response when they are successful, and a `400 Bad Request` error when
+there is any error interpreting the request.  For more visibility when your webhooks
+fail, see the [Troubleshooting](#troubleshooting) section below.
+
+## Webhook templates
+
+The purpose of a webhook is to accept an HTTP request from another system and produce a
+Prefect event from it.  You may find that you often have little influence or control
+over the format of those requests, so Prefect's webhook system gives you full control
+over how you turn those notifications from other systems into meaningful events in your
+Prefect Cloud workspace.  The template you define for each webhook will determine how
+individual components of the incoming HTTP request become the event name and resource
+labels of the resulting Prefect event.
+
+As with the [templates available in Prefect Cloud Automation](/cloud/automations) for
+defining notifications and other parameters, you will write templates in
+[Jinja2](https://jinja.palletsprojects.com/en/3.1.x/templates/).  All of the built-in
+Jinja2 blocks and filters are available, as well as the filters from the
+[`jinja2-humanize-extensions`
+package](https://pypi.org/project/jinja2-humanize-extension/).
+
+Your goal when defining your event template is to produce a valid JSON object that
+defines (at minimum) the `event` name and the `resource["prefect.resource.id"]`, which
+are required of all events.  The simplest template is one in which these are statically
+defined.
+
+### Static webhook events
+
+Let's explore an example for static webhook templates.  Pretend that you are configuring
```

Comment:
@discdiver, if you've got any advice on how to adjust the voice in this PR I'd appreciate it.  @WillRaphaelson's got a great point, that is is kind of "dungeon-mastery", like "imagine you are in a tavern...".

I wanted to walk through a concrete scenario because I think it helps to illustrate the use cases, but I don't see any other examples in the docs where we do that.